cobalt/site/docs/development/setup-android.md

```diff
@@ -3,10 +3,9 @@ Book: /youtube/cobalt/_book.yaml
 
 # Set up your environment - Android
 
-These instructions explain how to set up Cobalt for your workstation and Android
-device. The package being built here is referred to as CoAT (Cobalt on Android TV).
+These instructions explain how to build Cobalt with evergreen enabled for your workstation and run the api on Android device.
 
-## Preliminary Setup
+## Build Instruction
```

Comment:
Let's also add guidance for how to enable EG-Lite via the flag as there are legacy platforms that still use Lite.

Reply:
EG-Lite build and launch commands are updated. please help to check it.

Reply:
LGTM, thanks!